tests/backup/backup_license_test.go

```diff
@@ -105,36 +110,208 @@ var _ = Describe("{NodeCountForLicensing}", func() {
+				err = CreateBackupLocation(provider, bkpLocationName, backupLocationUID, cloudCredName, cloudCredUID, getGlobalBucketName(provider), orgID, "")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating backup location %s", bkpLocationName))
+			}
+		})
+		Step("Getting the number of worker nodes in source cluster and applying label portworx.io/nobackup=true to all its worker nodes", func() {
+			log.InfoD("Getting the total number of worker nodes in source cluster")
+			sourceClusterWorkerNodes = node.GetWorkerNodes()
+			log.InfoD("Total number of worker nodes in source cluster are %v", len(sourceClusterWorkerNodes))
+			totalNumberOfWorkerNodes = append(totalNumberOfWorkerNodes, sourceClusterWorkerNodes...)
+
+			log.InfoD("Applying label portworx.io/nobackup=true to all the worker node on source cluster")
+			for _, workerNode := range sourceClusterWorkerNodes {
+				err := Inst().S.AddLabelOnNode(workerNode, "portworx.io/nobackup", "true")
+				log.FailOnError(err, fmt.Sprintf("Failed to apply label portworx.io/nobackup=true to source cluster worker node %v", workerNode.Name))
+			}
+		})
+		Step("Getting the number of worker nodes in destination cluster and applying label portworx.io/nobackup=true to all its worker nodes", func() {
+			log.InfoD("Switching cluster context to destination cluster")
+			err := SetDestinationKubeConfig()
+			log.FailOnError(err, "Switching context to destination cluster failed")
+			log.InfoD("Getting the total number of worker nodes in destination cluster")
+			destinationClusterWorkerNodes = node.GetWorkerNodes()
+			log.InfoD("Total number of worker nodes in destination cluster are %v", len(destinationClusterWorkerNodes))
+			totalNumberOfWorkerNodes = append(totalNumberOfWorkerNodes, destinationClusterWorkerNodes...)
+			log.InfoD("Total number of worker nodes in source and destination cluster are %v", len(totalNumberOfWorkerNodes))
+			log.InfoD("Applying label portworx.io/nobackup=true to all the worker node on destination cluster")
+			for _, workerNode := range destinationClusterWorkerNodes {
+				err := Inst().S.AddLabelOnNode(workerNode, "portworx.io/nobackup", "true")
+				log.FailOnError(err, fmt.Sprintf("Failed to apply label portworx.io/nobackup=true to destination cluster worker node %v", workerNode.Name))
 			}
+			log.InfoD("Switching cluster context to source cluster")
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster failed")
+		})
+		Step("Verifying the license count before adding source and destination clusters", func() {
+			log.InfoD("Verifying the license count before adding source and destination clusters")
+			err = VerifyLicenseConsumedCount(ctx, orgID, 0)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying that license count before adding source and destination cluster to backup should be 0"))
+		})
+		Step("Registering source and destination clusters for backup", func() {
```

Comment:
"Registering source and destination ..." no longer makes sense in Step description.

Reply:
done